linkml/generators/docgen/class.md.jinja2

```diff
@@ -32,17 +32,16 @@ URI: {{ gen.uri_link(element) }}
 
 | Name | Cardinality and Range | Description | Inheritance |
 | ---  | --- | --- | --- |
-{% for s in schemaview.class_induced_slots(element.name) -%}
-{% if s.name in gen.get_direct_slots(element) -%}
-| {{gen.link(s)}} | {{ gen.cardinality(s) }} <br/> {{ gen.link(s.range) }} | {{ s.description }}  | direct |
+{%- if gen.get_direct_slots(element)|length > 0 %}
+{% for d_slot in gen.get_direct_slots(element) -%}
+| {{ gen.link(schemaview.get_slot(d_slot)) }} | {{ gen.cardinality(schemaview.get_slot(d_slot)) }} <br/> {{ schemaview.get_slot(d_slot).range }} | {{ schemaview.get_slot(d_slot).description }} | direct |
+{% endfor -%}
 {% endif -%}
-{%- if s.name in gen.get_indirect_slots(element) -%}
-| {{gen.link(s)}} | {{ gen.cardinality(s) }} <br/> {{ gen.link(s.range) }} | {{ s.description }}  | inherited |
+{% if gen.get_indirect_slots(element)|length > 0 -%}
+{% for i_slot in gen.get_indirect_slots(element) -%}
+| {{ gen.link(schemaview.get_slot(i_slot)) }} | {{ gen.cardinality(schemaview.get_slot(i_slot)) }} <br/> {{ schemaview.get_slot(i_slot).range }} | {{ schemaview.get_slot(i_slot).description }} | induced <br/> Class: _{{ gen.get_slot_inherited_from(element.name, i_slot)|join(', ') }}_ |
```

Comment:
You can get rid of `induced <br/> Class:`

And why not just make a docgen method that is the equivalent of link for lists of names?

Reply:
Done. I've added a method called `links()` in the documentation generator. I'll merge the PR in once the tests go through.